### ISSUE TITLE
Modify derivatives softsegs in `spine-generic/data-multi-subject` in `sb/add_extra_manual_seg`

### ISSUE THREAD

**Issue by sandrinebedard** [neuropoly/data-management#182]
## Description 
We modified the registration algorithm to create the soft segmentations (_softseg) in `spine-generic/data-multi-subject/derivatives/labels_softseg`. I uploaded the modified soft segmentations and added some extra manual segmentations in the branch `sb/add_extra_manual_seg`.

**Comment by mguaypaq** [comment]
Maybe this should have been a pull request on the `spine-generic/data-multi-subject` repository? Anyway, I started looking into this, and I'm getting a ton of merge conflicts, so I'll have to sort those out.

According to `git merge-base master origin/sb/add_extra_manual_seg`, the `sb/add_extra_manual_seg` branches off from `master` at this commit:
```
commit 082dd0d7b5ad1eedf53ea9765910e2d68769d355
Author: Alexandru Foias <alexfoias@users.noreply.github.com>
Date:   Tue Apr 12 10:50:06 2022 -0400
```

The `master` branch contains 4 more commits, but this one seems to be the most relevant one in terms of conflicts:
```
commit 33b8d1e8cf40a7d57ba8ed2ea3c80a9090b1851f
Author: Sandrine Bédard <71230552+sandrinebedard@users.noreply.github.com>
Date:   Tue Aug 30 12:12:52 2022 -0400

    Move derivatives from spine-generic-processed (#123)
    
    * add derivatives from spine-generic-processed/derivatives
    
    * add softsegs
    
    * remove old seg with suffixes
    
    * remove gmseg with suffix rms
    
    * remove old segmentations fro dwi
    
    * remove json file with old segmentation
    
    * remove wrongly named
    
    * update segmentation and json
    
    * add new manual segmentation and json
    
    * remove raw image in the derivatives
    
    * modify manual segmentations
    
    * update softsegs & add manual seg
    
    * add missing manual segmentation
    
    * add extra ew manual seg and softsegs
    
    * add missing author field
    
    * add missing coma
    
    * add manual seg of t2star
    
    Co-authored-by: Mathieu Guay-Paquet <mathieu.guaypaquet@gmail.com>
```

The branch `sb/add_extra_manual_seg` contains 20 more commits. It looks like the contents of 17 of them make up the above squash-and-merge commit, so that would explain why there are so many conflicts:
```
commit f78857c273b1b3b064a68359c84131ffb8284bda (HEAD -> wip, origin/sb/add_extra_man
ual_seg)
Author: Sandrine Bedard <sandrine.bedard@polymtl.ca>
Date:   Tue Nov 15 11:30:35 2022 -0500

    update README of labels_softseg

commit 736f83f4e53a3a1de46852174ef4cc654f8dec65
Author: Sandrine Bedard <sandrine.bedard@polymtl.ca>
Date:   Sat Oct 22 15:28:27 2022 -0400

    add new softsegs with reg centerofmass

commit 5ffe8f952503d69f5ed0ada65a402b41130588fa
Author: Sandrine Bedard <sandrine.bedard@polymtl.ca>
Date:   Thu Oct 6 11:02:07 2022 -0400

    add new manual segmentations

commit f0f462a4132b55f8788937b26cd5f4209c74d6cd
Author: Sandrine Bedard <sandrine.bedard@polymtl.ca>
Date:   Mon Aug 22 10:19:56 2022 -0400

    add manual seg of t2star

commit 2bf0cd6e8ce024d4480385808fc2537e4bebebde
Author: Sandrine Bedard <sandrine.bedard@polymtl.ca>
Date:   Mon Aug 22 10:11:26 2022 -0400

    add missing coma

commit 285c5991129e6528239baa86b4dc680f4069bd6d
Author: Sandrine Bedard <sandrine.bedard@polymtl.ca>
Date:   Mon Aug 22 09:54:20 2022 -0400

    add missing author field

commit 77fd84945d42cbf8d3c90194ea25e9881c76655e
Author: Sandrine Bedard <sandrine.bedard@polymtl.ca>
Date:   Fri Aug 19 18:26:33 2022 -0400

    add extra ew manual seg and softsegs

commit f4c2899b136461dcc4bb63f87169c02c5ce5d307
Author: Sandrine Bedard <sandrine.bedard@polymtl.ca>
Date:   Fri Aug 19 16:56:27 2022 -0400

    add missing manual segmentation

commit e1ed580c3ce3297f30cfe9250cf6e07b19a5ef79
Author: Sandrine Bedard <sandrine.bedard@polymtl.ca>
Date:   Wed Aug 17 17:47:04 2022 -0400

    update softsegs & add manual seg

commit 35c249decb96c410d6bdedf8acc2fac02dae5086
Author: Sandrine Bedard <sandrine.bedard@polymtl.ca>
Date:   Tue Aug 2 09:30:24 2022 -0400

    modify manual segmentations

commit e7577966d629c871add79040b6e1cbb7a2c0baa9
Author: Sandrine Bedard <sandrine.bedard@polymtl.ca>
Date:   Thu Jul 7 17:15:46 2022 -0400

    remove raw image in the derivatives

commit d2f367e493d65bce097ee25bc00a374601873fd0
Author: Sandrine Bedard <sandrine.bedard@polymtl.ca>
Date:   Thu Jul 7 15:07:31 2022 -0400

    add new manual segmentation and json

commit 173e24b9c2e43798368321630247e0da7815cfb1
Author: Sandrine Bedard <sandrine.bedard@polymtl.ca>
Date:   Thu Jul 7 14:38:39 2022 -0400

    update segmentation and json

commit d297a0988fca6790a95c26326cc2633ff9a338c0
Author: Sandrine Bedard <sandrine.bedard@polymtl.ca>
Date:   Wed Jul 6 16:31:22 2022 -0400

    remove wrongly named

commit f8bbfe43fd3c33e5639e83945dc84cad027d8ce7
Author: Sandrine Bedard <sandrine.bedard@polymtl.ca>
Date:   Wed Jul 6 14:09:03 2022 -0400

    remove json file with old segmentation

commit f88f3718cfb22be79519f4c0bde980f9fc100613
Author: Sandrine Bedard <sandrine.bedard@polymtl.ca>
Date:   Wed Jul 6 13:56:05 2022 -0400

    remove old segmentations fro dwi

commit e90f716476976b5d0a31abbbe4fbda4e6071f0e1
Author: Sandrine Bedard <sandrine.bedard@polymtl.ca>
Date:   Mon Jul 4 11:29:00 2022 -0400

    remove gmseg with suffix rms

commit 283d4ece3a39aff82309da5cad4df4c71f223655
Author: Sandrine Bedard <sandrine.bedard@polymtl.ca>
Date:   Mon Jul 4 11:27:25 2022 -0400

     remove old seg with suffixes

commit 9e2475cc1edfb5dbd0006d3f4d1a04462fa459ee
Author: Sandrine Bedard <sandrine.bedard@polymtl.ca>
Date:   Mon Jul 4 11:03:53 2022 -0400

    add softsegs

commit 52dfc9e6834b7f53742edf2fd1e72146bb731580
Author: Sandrine Bedard <sandrine.bedard@polymtl.ca>
Date:   Mon Jul 4 10:52:35 2022 -0400

    add derivatives from spine-generic-processed/derivatives
```

I'll get to work on fixing this. @sandrinebedard, when you have some time we should look at your copy of the repository together, to figure out how to avoid these kinds of conflicts in the future.

**Comment by sandrinebedard** [comment]
hello @mguaypaq ! Next time I'm at poly we can take a look at it together. If you want, I can open a PR in spine generic if you prefer? Thanks!

**Comment by mguaypaq** [comment]
This PR here will be fine, we can just use PRs in spine-generic in the future.

**Comment by mguaypaq** [comment]
Alright! I think I extracted the new commits correctly to a new branch, `mgp/fix`, and I'll open a PR on spine-generic (because the `master` branch is protected) (it's now [data-multi-subject#133](https://github.com/spine-generic/data-multi-subject/pull/133)). Here's what I ended up doing:

```bash

$ git clone git@github.com:spine-generic/data-multi-subject.git
$ cd data-multi-subject

$ # The commit range below seems to be the issue
$ git log -4 --format=short 33b8d1e8cf40a7d57ba8ed2ea3c80a9090b1851f
commit 33b8d1e8cf40a7d57ba8ed2ea3c80a9090b1851f
Author: Sandrine Bédard <71230552+sandrinebedard@users.noreply.github.com>

    Move derivatives from spine-generic-processed (#123)

commit 321798b270a210d1ccfa28ef0cbba8158f924d06
Author: Mathieu Guay-Paquet <mathieu.guaypaquet@gmail.com>

    Add instruction for working from a fork (#129)

commit 42e58865e3f26266d6b9899518942e317c85721f
Author: mbondy023 <98548430+mbondy023@users.noreply.github.com>

    add brain_t1 category into exclude.yml file (#127)

commit 082dd0d7b5ad1eedf53ea9765910e2d68769d355
Author: Alexandru Foias <alexfoias@users.noreply.github.com>

    Fix defaced images (#119)

$ # We can check that the change from the single commit
$ # "$new_base".."$squashed"
$ # should be the same as the change from the 17 commits
$ # "$old_base".."origin/sb/add_extra_manual_seg~3"
$ squashed=33b8d1e8cf40a7d57ba8ed2ea3c80a9090b1851f
$ new_base=082dd0d7b5ad1eedf53ea9765910e2d68769d355
$ old_base=321798b270a210d1ccfa28ef0cbba8158f924d06
$ git rebase --onto "$old_base" \
                    "$new_base" \
                    origin/sb/add_extra_manual_seg~3
$ git diff "$squashed"
$ # Since 'git diff' produced no output, they are indeed the same

$ # Now let's move the last 3 commits
$ # from sb/add_extra_manual_seg to master
$ git checkout -b mgp/fix --no-track origin/sb/add_extra_manual_seg
$ git rebase --onto master mgp/fix~3
$ git log -4 --format=short
commit 01762515bf8254d7c2954f2dce78831ff7bd0846 (HEAD -> mgp/fix)
Author: Sandrine Bedard <sandrine.bedard@polymtl.ca>

    update README of labels_softseg

commit c1d805f9341e1f885620f6450be25542ee33a43d
Author: Sandrine Bedard <sandrine.bedard@polymtl.ca>

    add new softsegs with reg centerofmass

commit 232c0bd9aa402a661737964ac2f64c31233d2d7c
Author: Sandrine Bedard <sandrine.bedard@polymtl.ca>

    add new manual segmentations

commit c0b2ba141e74495f03106ba5779aac2f7464b58f (origin/master, origin/HEAD, master)
Author: Julien Cohen-Adad <jcohen@polymtl.ca>

    Update README.md (#130)

$ # Check that bids-validator is happy
$ git config annex.private true
$ git config remote.origin.annex-readonly true
$ git annex get
$ bids-validator . --config.ignore=38 --config.ignore=39 --config.ignore=101
bids-validator@1.8.9
bids-specification@disable
(node:25037) Warning: Closing directory handle on garbage collection
(Use `node --trace-warnings ...` to show where the warning was created)
This dataset appears to be BIDS compatible.
        Summary:                  Available Tasks:        Available Modalities: 
        2698 Files, 8.51GB                                MRI                   
        267 - Subjects                                                          
        1 - Session                                                             


        If you have any questions, please post on https://neurostars.org/tags/bids.

$ # Upload the new branch to github, to open a PR
$ git push --set-upstream origin mgp/fix
```
